docs/h2t-scc.md

```diff
@@ -0,0 +1,187 @@
+# Troubleshoot Security Context Constraints (SCC)
+
+This document contains information regarding troubleshooting SCC related issues. Document includes the initial diagnosis step which helps find the actual error. To solve the SCC related issues need a basic understanding of the SCC and this is done by explaining what is SCC is along with its role in the cluster. 
+
+## Initial diagnosis:
+
+* `# oc describe pod <pod-name> | grep scc`
+
+This command show type of SCC used in the pod deployment. Knowing  SCC name will let us know about control permissions for a pod. These permissions include actions that a pod, a collection of containers, can perform, and what resources it can access. 
```

Comment:
```suggestion
* $ `oc describe pod <pod-name> | grep scc`

    This command show type of SCC used in the pod deployment. 
    Knowing  SCC name will let us know about control permissions for a pod. 
    These permissions include actions that a pod, a collection of containers, can perform, and what resources it can access. 
```
The `oc` cmd should not require or imply root user (by shell prompt).